packages/aor-graphql-client-graphcool/src/buildGqlQuery.js

```diff
@@ -34,19 +36,19 @@ export const buildFields = introspectionResults => fields =>
     }, {});
 
 export const getArgType = arg => {
-    if (arg.type.kind === TypeKind.NON_NULL) {
-        return `${arg.type.ofType.name}!`;
-    }
+    const type = getFinalType(arg.type);
+    const required = isRequired(arg.type);
+    const list = isList(arg.type);
 
-    return arg.type.name;
+    return `${list ? '[' : ''}${type.name}${list ? '!]' : ''}${required ? '!' : ''}`;
 };
 
 export const buildArgs = (query, variables) => {
     if (query.args.length === 0) {
         return {};
     }
 
-    const validVariables = Object.keys(variables).filter(k => !!variables[k] && variables[k] !== null);
+    const validVariables = Object.keys(variables).filter(k => typeof variables[k] !== 'undefined');
```

Comment:
Shouldn't we check for null too here ?

Reply:
When you set relation to null this check doesn't let the arg propagate, thus causing errors from Graphcool.